src/main/scanLogic/scanRunners/analyzerModels.ts

```diff
@@ -0,0 +1,66 @@
+export interface AnalyzerRequest {
+    scans: AnalyzeScanRequest[];
+}
+
+type AnalyzerType = 'analyze-applicability' | 'analyze-codebase';
+
+export interface AnalyzeScanRequest {
+    // What type of scan
+    type: AnalyzerType; 
+    // The path that the response will be written to
+    output: string; 
+    // List of path to folders that scan will run inside
+    roots: string[]; 
+}
+
+export interface AnalyzerScanResponse {
+    runs: AnalyzerScanRun[];
+}
+
+export interface AnalyzerScanRun {
+    tool: {
```

Comment:
Implicit typing should be avoided

Reply:
there is no use to it, for now it has only one attribute with no uses of holding the middle object